### DISCUSSION TITLE
OpenAI cybersecurity grant program

### DISCUSSION THREAD

**Discussion by hyoklee** [aldanor/hdf5-rust#242] [Ideas]
I think this project has a chance for this grant:

https://openai.com/blog/openai-cybersecurity-grant-program

**Comment by aldanor** [comment]
🤔 Thanks for raising this @hyoklee but how is HDF5 related to cybersec/netsec? (especially the Rust crate around it, that is)

**Comment by hyoklee** [comment]
The proposal call has

`Help developers port code to memory safe languages`

It also mentions this report that recommends Rust:

https://www.cisa.gov/sites/default/files/2023-04/principles_approaches_for_security-by-design-default_508_0.pdf
